Classic_RPG/bin/.gitignore

```diff
@@ -1 +1,3 @@
+/entities/
+/game_management/
```

Comment:
What is that gitignore and why is he ignoring entities and game_management package ?